ci/prow/config.yaml

```diff
@@ -1789,6 +1953,68 @@ periodics:
       - "--cov-target=./apis/ ./configmap/"
       - "--cov-threshold-percentage=80"
 
+- cron: "30 * * * *" # Run every hour and 30 minutes
+  name: ci-knative-caching-continuous
+  agent: kubernetes
+  labels:
+    preset-service-account: "true"
+    preset-bazel-scratch-dir: "true"
+    preset-bazel-remote-cache-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/knative-tests/test-infra/prow-tests:latest
+      imagePullPolicy: Always
+      args:
+      - "--ssh=/etc/ssh-knative/ssh-knative"
+      - "--scenario=kubernetes_execute_bazel"
+      - "--clean"
+      - "--job=$(JOB_NAME)"
+      - "--repo=github.com/knative/caching"
+      - "--root=/go/src"
+      - "--service-account=/etc/service-account/service-account.json"
+      - "--upload=gs://knative-prow/logs"
+      - "--timeout=50" # Avoid overrun
+      - "--" # end bootstrap args, scenario args below
+      - "--" # end kubernetes_execute_bazel flags (consider following flags as text)
+      - "./test/presubmit-tests.sh"
+      - "--all-tests"
+      # Bazel needs privileged mode in order to sandbox builds.
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          memory: "1Gi"
+      volumeMounts:
+      - mountPath: /etc/ssh-knative
+        name: ssh-knative
+    volumes:
+     - name: ssh-knative
+       secret:
+         defaultMode: 256
+         secretName: ssh-knative
+- cron: "0 1 * * *" # Run at 01:00 every day
+  name: ci-knative-caching-go-coverage
+  agent: kubernetes
+  decorate: true
+  ssh_key_secrets:
+  - ssh-knative
+  extra_refs:
+  - org: knative
+    repo: caching
+    base_ref: master
+    clone_uri: "git@github.com:knative/caching.git"
+  spec:
+    containers:
+    - image: gcr.io/knative-tests/test-infra/coverage:latest
+      imagePullPolicy: Always
+      command:
+      - "/coverage"
+      args:
+      - "--artifacts=$(ARTIFACTS)"
+      - "--profile-name=coverage_profile.txt"
+      - "--cov-target=./pkg/"
+      - "--cov-threshold-percentage=80"
```

Comment:
80 might be too high for a new repo?

Reply:
It's not a blocker, so I don't think it's a problem.

Reply:
@steuhs, I see threshold parameter in post, continuous, pull coverage jobs. Which job will block PR, is it pull? If yes, 
what's the threshold usage in other post, continuous jobs?

Reply:
And actually ALL repos have this threshold.